lambda_src/drivers/process_boto3.py

```diff
@@ -3,21 +3,53 @@
 import boto3
 from botocore.response import StreamingBody
 
+from utils import pick
 
 DISALLOWED_CLIENTS = {'kms', 'secretsmanager'}
 
 
 def process_row(
-    client_name, method_name, results_path=None, region='us-west-2', **kwargs
+    client_name,
+    method_name,
+    assume_role_arns=None,
```

Comment:
Is this always ARNs? Is it always a list? What possible values can this variable take?

If it is a sequence of roles that need to be assumed would it make sense to add the keyword `sequence` or `chain` in the variable to indicate that?

Reply:
it's a sequence of roles, yes, with either strings or objects which are going to be passed into `assume_role`, e.g. `{"RoleArn":"...","ExternalId":"..."}`